libs/langchain/langchain/chains/loading.py

```diff
@@ -114,17 +114,55 @@ def _load_map_reduce_documents_chain(
     if not isinstance(llm_chain, LLMChain):
         raise ValueError(f"Expected LLMChain, got {llm_chain}")
 
-    if "combine_document_chain" in config:
+    if "reduce_documents_chain" in config:
+        reduce_documents_chain = load_chain_from_config(
+            config.pop("reduce_documents_chain")
+        )
+    elif "reduce_documents_chain_path" in config:
+        reduce_documents_chain = load_chain(config.pop("reduce_documents_chain_path"))
+    else:
+        reduce_documents_chain = _load_reduce_documents_chain(config)
```

Comment:
@hwchase17 I believe this line can handle older formats since it parses `combine_document_chain` from `config` instead of `reduce_documents_chain`. 

LMK, if you think differently, happy to adjust the PR for a backward-compatibility

Reply:
ah good call, i think this should work then